crates/circuit/src/parameter_table.rs

```diff
@@ -76,11 +77,30 @@ struct VectorInfo {
 /// `ParameterTable`.
 #[derive(Clone, Copy, Hash, PartialEq, Eq, Debug)]
 pub struct ParameterUuid(u128);
```

Comment:
`ParameterUuid` could be replaced by Rust's own `Uuid` object if we want. The only addition we really need here is the `from_parameter` method which is only called once in a debug method, so we could also move that there.